gis/mod_tile/Portfile

```diff
@@ -0,0 +1,252 @@
+    depends_run-append \
+        port:osmium-tool \
+        port:osmosis \
+        port:trim_osc
+}
+
+variant logrotate description {Logrotate configuration} {
+    depends_run-append port:logrotate
+}
+
+default_variants +osmosis +logrotate
+
+if {![variant_isset postgresql12] && ![variant_isset postgresql13]} {
+    default_variants-append +postgresql13
+}
+
+post-extract {
+    file rename ${worksrcpath}/renderd.conf ${worksrcpath}/renderd.conf.orig
+}
+
+post-patch {
+    xinstall -m 0644 -W ${filespath} \
+        renderd.conf.dist \
+        osm-tiles-update.conf.dist \
+        ${worksrcpath}
+    reinplace "s|@PREFIX@|${prefix}|g" \
+        ${worksrcpath}/renderd.conf.dist \
+        ${worksrcpath}/osmosis-db_replag
+}
+
+post-destroot {
+    system -W ${worksrcpath} "make install-mod_tile DESTDIR=${destroot} PREFIX=${prefix}"
+    if {[variant_isset osmosis]} {
+        xinstall -m 0755 -d ${destroot}${prefix}/etc/LaunchAgents/${agent_uniquename}
+        set minutes 20; # Run every ${minutes} minutes
+        set plist [open "${destroot}${prefix}/etc/LaunchAgents/${agent_uniquename}/${agent_uniquename}.plist" w 0644]
+        puts ${plist} {<?xml version="1.0" encoding="UTF-8"?>}
+        puts ${plist} {<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN"}
+        puts ${plist} {"http://www.apple.com/DTDs/PropertyList-1.0.dtd">}
+        puts ${plist} {<plist version="1.0">}
+        puts ${plist} {  <dict>}
+        puts ${plist} {    <key>Disabled</key>}
+        puts ${plist} {    <false/>}
+        puts ${plist} {    <key>UserName</key>}
+        puts ${plist} "    <string>${renderd_user}</string>"
+        puts ${plist} {    <key>Label</key>}
+        puts ${plist} "    <string>${agent_uniquename}</string>"
+        puts ${plist} {    <key>Program</key>}
+        puts ${plist} "    <string>${prefix}/share/mod_tile/openstreetmap-tiles-update-expire</string>"
+        puts ${plist} {    <!-- See man launchd.plist -->}
+        puts ${plist} {    <key>StartCalendarInterval</key>}
+        puts ${plist} {    <array>}
+        for {set i [expr int(rand() * $minutes)]} {$i < 60} {incr i $minutes} {
+            puts ${plist} {        <dict>}
+            puts ${plist} {            <key>Minute</key>}
+            puts ${plist} "            <integer>${i}</integer>"
+            puts ${plist} {        </dict>}
+        }
+        puts ${plist} {    </array>}
+        puts ${plist} {    <key>ProcessType</key>}
+        puts ${plist} {    <string>Background</string>}
+        # puts ${plist} {    <!--}
+        # puts ${plist} {    <key>WorkingDirectory</key>}
+        # puts ${plist} "    <string>${prefix}/share/renderd</string>"
+        # puts ${plist} {    -->}
+        # puts ${plist} {    <!--}
+        # puts ${plist} {    <key>StandardOutPath</key>}
+        # puts ${plist} "    <string>${prefix}/var/log/renderd/update-expire.log</string>"
+        # puts ${plist} {    -->}
```

Comment:
are these commented out lines needed for anything, if not please remove

Reply:
The commented out lines are not needed.  I’ve removed them.